tests/unit_tests/tests_launch/test_launch.py

```diff
@@ -0,0 +1,36 @@
+import os
+import time
+
+import wandb
+from wandb.sdk.internal.internal_api import Api as InternalApi
+from wandb.sdk.launch.launch_add import launch_add
+
+
+def test_launch_delete_queued_run(relay_server, runner, user, monkeypatch):
+    queue = "default"
+    proj = "test"
+    uri = "https://github.com/gtarpenning/wandb-launch-test"
```

Comment:
What is a good public project to use as a test URI? It has to be accessible to the test-container user, and the wandb.ai URI's I would otherwise use don't seem to work...

Reply:
I think wandb-examples is the right one to use